common/djangoapps/util/views.py

```diff
@@ -383,6 +384,13 @@ def get_feedback_form_context(request):
 
     context["support_email"] = configuration_helpers.get_value('email_from_address', settings.DEFAULT_FROM_EMAIL)
 
+    #Update the tag info with 'enterprise_learner' if the user belongs to an enterprise customer.
+    enterprise_learner_data = enterprise_api.get_enterprise_learner_data(site=request.site, user=request.user)
+    if enterprise_learner_data:
+        enterprise_customer_name = enterprise_learner_data[0]['enterprise_customer']['name']
+        context["tags"]["learner_type"] = "enterprise_learner"
+        context["additional_info"]["enterprise_customer_name"] = enterprise_customer_name
```

Comment:
This is correct. You need to add `enterprise_customer_name` to `_get_zendesk_custom_field_context`, and a custom field will need to be added to both the edX Zendesk instance and mapped using the `ZENDESK_CUSTOM_FIELDS` setting.